include/decay.hpp

```diff
@@ -15,7 +15,7 @@ namespace hypha {
         uint64_t newPeriod;
     };
 
-    const DecayResult decay(
+    const DecayResult Decay(
```

Comment:
Are we following any naming standard / strategy ?

Reply:
I did not think I changed that...

Reply:
@IanMendozaJaimes

Reply:
oh sorry for that, I was trying to see if that was causing my compilation error. I think I will change it again